app/uk/gov/hmrc/tariffclassificationfrontend/connector/BindingTariffClassificationConnector.scala

```diff
@@ -46,4 +46,9 @@ class BindingTariffClassificationConnector @Inject()(configuration: AppConfig, c
     client.GET[Seq[Case]](url)
   }
 
+  def updateCase(c: Case)(implicit hc: HeaderCarrier): Future[Option[Case]] = {
```

Comment:
Should an update method return an `Option`?

I'd expect this to just return a `Case`